components/expenses/ExpenseMoreActionsButton.js

```diff
@@ -104,6 +115,14 @@ const ExpenseMoreActionsButton = ({ expense, collective, onError, onEdit, isDisa
                 <FormattedMessage id="Edit" defaultMessage="Edit" />
               </Action>
             )}
+            {permissions?.canDelete && (
+              <DeleteExpenseButton
```

Comment:
The other actions use the `Action` component, which guarantees all links in this menu keep the same styles. Could we re-use that rather than re-defining all the styles in `StyledDeleteButton`?